### ISSUE TITLE
Dark Theme: Caps lock icon is hardly visible

### ISSUE THREAD

**Issue by coderste** [desktop/desktop#5745]
## Description
When signing into the GitHub account it ask for your password. On the password field if you have the caps lock on it will show you that you have caps lock enabled on the input field but this box is not very visible on the dark theme.

## Version
Version 1.4.0

* GitHub Desktop:
* Operating system:
macOS Sierra
Version: 10.12.6

## Steps to Reproduce
Github Desktop -> Preferences and then the Accounts tab.

### Attached Screenshot
![image](https://user-images.githubusercontent.com/13105865/46074861-adc2ce80-c180-11e8-94a0-61a610f231bd.png)

Is there anyway we could inverse this color for the dark theme or is that not possible?

**Comment by shiftkey** [comment]
@coderste thanks for the feedback!

That bit of UI is actually macOS-specific (I don't see the same prompt on Windows) and I don't see any DOM changes when interacting with the field that we might be able to target and re-style from in the app. A quick Google scan also didn't yield many clues or other cases about styling this. 🤔

**Comment by coderste** [comment]
@shiftkey Yeah I had a feeling that maybe you guys wasn't able to handle that - but with the next macOS release bringing dark mode to a few of there native apps maybe they have added the ability to change the color of that? Not sure

**Comment by j-f1** [comment]
I think there is some sort of pseudo-selector for it, but that might be Safari-specific.

**Comment by iamnapo** [comment]
Hello,

I'd like to work on this 😃

**Comment by shiftkey** [comment]
@iamnapo no problem - good luck, let us know how you go!

**Comment by tierninho** [comment]
Not much online when searching for a solution, however these might help or offer clues:

https://medium.com/@tbarrasso/html5-input-attributes-pseudo-elements-12b2f94b1711
https://stackoverflow.com/questions/13238893/disable-caps-lock-indicator-on-webkit-browsers
https://github.com/mui-org/material-ui/issues/4119

Good luck to the one who picks this up! 😄

**Comment by j-f1** [comment]
I wonder if setting the text color back to black and `filter: invert()`-ing the input would work.

**Comment by billygriffin** [comment]
Since this hasn't been something we've seen come up as an issue since the original report and because there hasn't been anyone interested in picking it up, I'm going to close it, but if someone comes along and has a straightforward solution for it, please feel free to respond here and we're not opposed to reopening. Thanks!

**Comment by j-f1** [comment]
This still reproduces

<img width="470" alt="Screen Shot 2019-11-22 at 21 38 31" src="https://user-images.githubusercontent.com/25517624/69471716-6e861280-0d70-11ea-947c-33e5582d50e6.png">

and I’ve confirmed switching to light mode styles for the text box and inverting it keeps the indicator visible:

<img width="425" alt="Screen Shot 2019-11-22 at 21 40 56" src="https://user-images.githubusercontent.com/25517624/69471747-e6543d00-0d70-11ea-8798-dcfc21d999ef.png">
